### PR TITLE
[RW-4888][risk=no] Add helptext to notebooks

### PR DESCRIPTION
**Pull request by als364**
This text is now on both the notebook preview page and the actual notebook page.
Screen Shot 2020-05-04 at 3 38 21 PM

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally

### CODE REVIEW

File: ui/src/app/pages/workspace/workspace-wrapper/component.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h3 class="modal-title">Error:</h3>
                     [shareFunction]="handleShareAction"
                     [sidebarOpen]="sidebarOpen"
                     [notebookStyles]="notebookStyles"></app-help-sidebar>
-  <div [style.margin-right]="helpContent ? '45px' : 0" [style.height]="helpContent ? 'auto': '100%'">
+  <div [style.margin-right]="helpContent ? '45px' : 0" [style.height]="helpContent && !notebookRouterOutletContainerStyles ? 'auto': '100%'">
```
Comment (calbach) [inline ui/src/app/pages/workspace/workspace-wrapper/component.html:32]: Please add a comment somewhere to explain what this is doing, or else rename property to be more clearly tied to the display implications per above comment

File: ui/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -142,6 +142,9 @@ const routes: Routes = [
                       // use the (urldecoded) captured value nbName
                       pathElementForTitle: 'nbName',
                       breadcrumb: BreadcrumbType.Notebook,
+                      helpContent: 'notebookStorage',
+                      notebookHelpSidebarStyles: true,
+                      notebookRouterOutletContainerStyles: true,
```
Comment (calbach) [inline ui/src/app/app-routing.module.ts:147]: Is there a more semantic name that can be used here, e.g. "notebookFullscreenStyle" (or something similar - I don't follow the goal for the property). I'm not following how this relates to a router outlet.

File: ui/src/app/app-routing.module.ts [hunk 3]
```diff
@@ -150,7 +153,7 @@ const routes: Routes = [
                     data: {
                       pathElementForTitle: 'nbName',
                       breadcrumb: BreadcrumbType.Notebook,
-                      helpContent: 'preview',
+                      helpContent: 'notebookStorage',
```
Comment (calbach) [inline ui/src/app/app-routing.module.ts:156]: Perhaps this would be explained by the other comments, but why is the new property not set here?

Comment (als364) [reply ui/src/app/app-routing.module.ts:156]: Screen Shot 2020-05-04 at 3 02 55 PM
`height: auto` behaves strangely when we're actually displaying the notebook. The content of the div is large enough that it ought to make the container div grow, but it's not doing that.

File: ui/src/app/app-routing.module.ts (hunk 2 above)
Comment (als364) [reply ui/src/app/app-routing.module.ts:147]: How about something like `fullHeightOverride`?

Comment (calbach) [reply ui/src/app/app-routing.module.ts:147]: OK I see what it's doing now. Maybe `containerFullHeightOverride` or `contentFullHeightOverride`? 

Comment in workspace-wrapper wouldn't hurt as well to explain that it's needed primarily due to the notebooks page iframe

**Pull request merged**